js_functions/task3.js

```diff
@@ -0,0 +1,9 @@
+function factorial(num){
```

Comment:
it is not `factorial`. this name will make anyone confused. Use more informative name like `countDown`